### PR TITLE
Update team data

### PR DESCRIPTION
**Pull request by hanakoori01**
### ¿Qué hace este PR?

- Resuelve #135 

### Pasos 
1. Correr `yarn start`
2. Ir a Equipo

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

File: src/pages/equipo.js [hunk 1]
```diff
@@ -1,386 +1,413 @@
...
+    color: "#ffffff",
+  },
+  {
+    img: "img/team/jorge.jpg",
+    name: "Jorge Murillo",
+    position: "Diseñador UX Sr.",
+    bio: "Entender las necesidades es crucial para crear interfases para apps de alta tecnología y mediar entre usuarios, diseñadores y desarrolladores.",
+    linkedIn: "https://www.linkedin.com/in/jmurilloux/",
+    twitter: "https://twitter.com/jmur1ll0",
+    color: "#ffffff",
```
Comment (AngeloCG97) [inline src/pages/equipo.js:107]: Please review mobile version, color should be alternated between #f8f8f8 and #ffffff

File: src/pages/equipo.js [hunk 2]
```diff
@@ -1,386 +1,413 @@
...
+            <Box className="spacingBox">
+              <h2>Equipo gerencial</h2>
+            </Box>
+            <Box>
+              <Grid container>
+                {ManagementTeam.map((person) => (
+                  <Grid item xs={12} md={6} key={person.name}>
+                    <Box
+                      className="teamPadding"
+                      style={{ backgroundColor: isMobile ? person.color : "" }}
```
Comment (AngeloCG97) [inline src/pages/equipo.js:253]: Please change style={{ backgroundColor: isMobile ? person.color : "" }} to bgColor Box prop

File: src/pages/equipo.js [hunk 3]
```diff
@@ -1,386 +1,413 @@
...
+            <Box className="spacingBox">
+              <h2>Mercadeo y ventas</h2>
+            </Box>
+            <Box>
+              <Grid container>
+                {MarketingSalesTeam.map((person) => (
+                  <Grid item xs={12} md={6} key={person.name}>
+                    <Box
+                      className="teamPadding"
+                      style={{ backgroundColor: isMobile ? person.color : "" }}
```
Comment (AngeloCG97) [inline src/pages/equipo.js:279]: Please change style={{ backgroundColor: isMobile ? person.color : "" }} to bgColor Box prop

File: src/pages/equipo.js [hunk 4]
```diff
@@ -1,386 +1,413 @@
...
+            <Box className="spacingBox">
+              <h2>Desarrollo y diseño</h2>
+            </Box>
+            <Box>
+              <Grid container>
+                {DevelopTeam.map((person) => (
+                  <Grid item xs={12} md={6} key={person.name}>
+                    <Box
+                      className="teamPadding"
+                      style={{ backgroundColor: isMobile ? person.color : "" }}
```
Comment (AngeloCG97) [inline src/pages/equipo.js:305]: Please change style={{ backgroundColor: isMobile ? person.color : "" }} to bgColor Box prop

File: src/pages/equipo.js [hunk 5]
```diff
@@ -1,386 +1,413 @@
...
+            <Box className="spacingBox">
+              <h2>Infraestructura y redes</h2>
+            </Box>
+            <Box>
+              <Grid container>
+                {InfrastructureNetworkingTeam.map((person) => (
+                  <Grid item xs={12} md={6} key={person.name}>
+                    <Box
+                      className="teamPadding"
+                      style={{ backgroundColor: isMobile ? person.color : "" }}
```
Comment (AngeloCG97) [inline src/pages/equipo.js:331]: Please change style={{ backgroundColor: isMobile ? person.color : "" }} to bgColor Box prop

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]
Please review comments

**Comment by hanakoori01** [comment]
> Please review comments

On it!

File: src/pages/equipo.js [hunk 6]
```diff
@@ -1,386 +1,413 @@
...
-                    <Box className="titleBox">
-                        <h1>Conozca al equipo</h1>
-                    </Box>
-                </Box>
-          }
+  const HeroSection = () => {
+    return (
+      <Box className="containerSec">
+        {isDesktop && (
+          <Box className="sectionHero" style={{ height: "auto" }}>
```
Comment (AngeloCG97) [inline src/pages/equipo.js:177]: Please remove this inline style code

File: src/pages/equipo.js [hunk 7]
```diff
@@ -1,386 +1,413 @@
...
+            <Box className="titleBox">
+              <h1>Conozca al equipo</h1>
+            </Box>
+            <TeamSection />
           </Box>
-        )
-    } 
+        )}
+        {isMobile && (
+          <Box className="sectionHeroMobile" style={{ paddingBottom: 0 }}>
```
Comment (AngeloCG97) [inline src/pages/equipo.js:185]: Please remove this inline style code, use pb box prop

File: src/pages/equipo.js [hunk 8]
```diff
@@ -1,386 +1,413 @@
...
-    }
+  const PersonCard = ({ img, name, position, bio, linkedIn, twitter }) => {
+    return (
+      <Grid container>
+        <Grid item xs={12} md={6}>
+          <Box className="avatarBox">
+            {img && (
+              <Avatar
+                variant="square"
+                style={{ width: "200px", height: "250px" }}
```
Comment (AngeloCG97) [inline src/pages/equipo.js:203]: remove inline style code

File: src/pages/equipo.js [hunk 9]
```diff
@@ -1,386 +1,413 @@
...
+                src={useBaseUrl(img)}
+              />
+            )}
+            {!img && (
+              <Avatar
+                variant="square"
+                style={{ width: "200px", height: "250px" }}
```
Comment (AngeloCG97) [inline src/pages/equipo.js:210]: remove inline style code

File: src/pages/equipo.js [hunk 10]
```diff
@@ -1,386 +1,413 @@
...
+          </Box>
+          <Box className="socialMediaBox">
+            {linkedIn && (
+              <a href={linkedIn} target="_blank" className="socialMediaIcon">
+                <LinkedInIcon style={{ width: "30px", height: "30px" }} />
+              </a>
+            )}
+            {twitter && (
+              <a href={twitter} target="_blank" className="socialMediaIcon">
+                <TwitterIcon style={{ width: "30px", height: "30px" }} />
```
Comment (AngeloCG97) [inline src/pages/equipo.js:230]: remove inline style code

File: src/pages/equipo.js [hunk 11]
```diff
@@ -1,386 +1,413 @@
...
+        <Grid item xs={12} md={6}>
+          <Box>
+            <h3 className="nameText">{name}</h3>
+            <p className="positionText">{position}</p>
+            <p className="bioText">{bio}</p>
+          </Box>
+          <Box className="socialMediaBox">
+            {linkedIn && (
+              <a href={linkedIn} target="_blank" className="socialMediaIcon">
+                <LinkedInIcon style={{ width: "30px", height: "30px" }} />
```
Comment (AngeloCG97) [inline src/pages/equipo.js:225]: remove inline style code

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]

**Pull request merged**